### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

Related to https://github.com/unlock-protocol/unlock/issues/11120, let's start to remove more un-used components and make some clean 

- [x] Remove `Drawer` in `unlock-app` and use the component from `UI`
- [x] `.jsx` -> `.tsx`
- [x] Small refactoring & cleanup 

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by searchableguy — APPROVED** [review]

File: unlock-app/src/hooks/useKeyManager.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ interface Options {
   enabled?: boolean
 }
 
+// todo: remove? not used anywhere
```
Comment (kalidiagne) [inline unlock-app/src/hooks/useKeyManager.ts:16]: @searchableguy wdyt? was introduces last month by you is that needed?

Comment (searchableguy) [reply unlock-app/src/hooks/useKeyManager.ts:16]: Yes. I wanted to to use it for stimulating.

**Comment by julien51** [comment]
Niiiice!

**Comment by kalidiagne** [comment]
can i have approval here?

**Review by julien51 — APPROVED** [review]

**Pull request merged**